doc/doxygen/scripts/program2doxygen

```diff
@@ -24,10 +24,11 @@ while ( m!^/\*!  ||  m!\s*\*! || m/^$/ ) {
     $_ = <>;
 }
 
-# have two states, in which the program can be:
-# comment-mode and program-mode
+# have three states, in which the program can be:
+# comment-mode, program-mode and skip-mode
```

Comment:
Can you submit this separately? I don't quite know what it's used for, but it seems independent of the rest.